### PR TITLE
Add transformation validation and support multiple columns using same directive

### PR DESCRIPTION
**Pull request by seanzhougoogle**
1. add transformation validations
2. support multiple column using same directive

### CODE REVIEW

File: delta-proto/src/main/java/io/cdap/delta/proto/DeltaConfig.java [hunk 1]
```diff
@@ -191,6 +192,13 @@ public Stage getTarget() {
     if (sourceStage == null) {
       throw new IllegalArgumentException("No source found.");
     }
+
+    if (tableTransformations != null) {
+      for (TableTransformation tableTransformation : tableTransformations) {
+        tableTransformation.validate();
+      }
+
```
Comment (sagarkapare) [inline delta-proto/src/main/java/io/cdap/delta/proto/DeltaConfig.java]: minor nit: remove extra line.

File: delta-proto/src/main/java/io/cdap/delta/proto/DeltaConfig.java [hunk 2]
```diff
@@ -191,6 +192,13 @@ public Stage getTarget() {
     if (sourceStage == null) {
       throw new IllegalArgumentException("No source found.");
     }
+
+    if (tableTransformations != null) {
```
Comment (sagarkapare) [inline delta-proto/src/main/java/io/cdap/delta/proto/DeltaConfig.java:196]: I think I missed in last PR review, but we should add `tableTransformations` to the equals and hashCode method as well.

**Review by sagarkapare — COMMENTED** [review]
minor comments, else LGTM 👍

**Review by sagarkapare — APPROVED** [review]

File: delta-proto/src/main/java/io/cdap/delta/proto/DeltaConfig.java (hunk 1 above)
Comment (seanzhougoogle) [reply delta-proto/src/main/java/io/cdap/delta/proto/DeltaConfig.java]: done :)

File: delta-proto/src/main/java/io/cdap/delta/proto/DeltaConfig.java (hunk 2 above)
Comment (seanzhougoogle) [reply delta-proto/src/main/java/io/cdap/delta/proto/DeltaConfig.java:196]: good catch :) Added equals and hashCode method for TableTransformations and ColumnTransformations too :)

**Pull request merged**